External/smoke/CMakeLists.txt

```diff
@@ -0,0 +1,572 @@
+# Smoke
+# https://github.com/ROCm-Developer-Tools/aomp.git
+
+include(External)
+
+option(TEST_SUITE_SMOKE_FORCE_ALL "Execute all Smoke tests, even those known to be unsupported by Clang" OFF)
+
+set(TEST_SUITE_OFFLOADING_FLAGS --offload-arch=native CACHE STRING "Compiler arguments for OpenMP offloading")
+set(TEST_SUITE_OFFLOADING_LDFLAGS --offload-arch=native CACHE STRING "Linker arguments for OpenMP offloading")
+
+set(INTEL_TEST_SUITE_EXPECT_PASS)
+
+set(AMD_TEST_SUITE_EXPECT_PASS
+    alignedattribute
+    already_locked
+    ancestorC
+    aomp-issue376
+    assertok_error
+    assume_no_openmp
+    atomic
+    atomic_double
+    big-jump-loop-1
+    big-jump-loop-collapse
+    big-jump-loop-env-vars
+    big-jump-loop-matmul
+    big-jump-loop-split-2
+    big-jump-loop-split-3
+    big-jump-loop-split-4
+    bugz-50967
+    bugz-50967-c
+    bugz-50968
+    CanonNE
+    check_smoke.sh
+    clang-262701
+    clang-273738
+    clang-273739
+    clang-274983
+    clang-282491-1
+    clang-282491-2
+    clang-282491-3
+    clang-296953-ornla-38
+    clang-310869-scan
+    clang-313307_simple_arr
+    clang-313307_simple_spmd
+    clang-325070
+    clang-326105
+    clang-337336
+    clang-357649
+    clang-358841
+    clang-359947
+    clang-373823
+    clang-383246
+    clang-388381
+    clang-392854
+    clang-alloc1
+    clang-gfx1030
+    clang-host-targ
+    clang-ifaces
+    clang-meta
+    clang-meta2
+    clang-meta3
+    clang-meta4
+    clang-meta5
+    clang-nest-nothing
+    clang-new-driver
+    clang_private
+    clang-red-DblComplex
+    clang-red-SglComplex
+    clang-reduction-and
+    clang-tracekernel
+    class
+    closed_bug2
+    collapse
+    collapse1
+    collapse-nonrectangle
+    collapse_parallel_spmd
+    complex
+    complex2
+    complex-debug
+    complex_reduction
+    const_global_ctor-308039
+    d2h_slow_copy
+    data_enter_issue01
+    data_issue_59
+    data_share1
+    data_share2
```

Comment:
This one for sure has a `printf` that is currently unsupported upstream, I believe.

Reply:
is this only for data_share2 or lines 83 to 86?